### DISCUSSION TITLE
Developing language not showing up

### DISCUSSION THREAD

**Discussion by jimmycabuy** [github-linguist/linguist#5878] [Classification] [answered]
Hi,

My developing languages are not displaying anymore, can someone give me an hint on how solve this problem ?? Thx !

https://github.com/jimmycabuy/LaCapsule

**Comment by lildude** [comment, marked as answer]
You're not seeing any analysis because your repo has a lot of files in it so it hitting [this hard limit](https://github.com/github/linguist/blob/c497388e7f88736d7a2fe2735f64366b41a4b5f8/lib/linguist/repository.rb#L132).

You'll need to reduce the number of files in your repo if you want langauge analysis to happen.

This is the same as discussed in https://github.com/github/linguist/discussions/5538 and https://github.com/github/linguist/discussions/5445 and https://github.com/github/linguist/discussions/5524

**Reply by jimmycabuy** [reply]
Thank you very much :)
